pkg/plugins/kms/AriesKMS.go

```diff
@@ -0,0 +1,27 @@
+package kms
+
+import "github.com/google/tink/go/keyset"
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable KMS service
+type Creator func(prov Provider) (AriesKMS, error)
+
+// AriesKMS is an interface defining a pluggable KMS operations provider to Aries agents.
+// It's responsible for managing keys and provide KMS specific crypto operations.
+type AriesKMS interface {
```

Comment:
Same comment as earlier - We shouldn't add prefixes.

Reply:
will remove prefix

Reply:
prefix removed